From 67fd2d1ce61e184b8fc4fdd21eb7eee3c375cc52 Mon Sep 17 00:00:00 2001
Message-ID: <67fd2d1ce61e184b8fc4fdd21eb7eee3c375cc52.1720653590.git.sharlatanus@gmail.com>
In-Reply-To: <6ebbf954f399ca8347144b087fae787c2b071354.1720653590.git.sharlatanus@gmail.com>
References: <6ebbf954f399ca8347144b087fae787c2b071354.1720653590.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Mon, 1 Jul 2024 21:46:20 +0100
Subject: [PATCH v2 2/4] gnu: python-dolfin-adjoint: Adjust package style.

* gnu/packages/simulation.scm (python-dolfin-adjoint): Move inputs after
arguments.
[description]: Place it on a new line, fix indentation.

Change-Id: I3d971e48b4072258ed0b41af5c202e64af9de8f3
---
 gnu/packages/simulation.scm | 40 ++++++++++++++++++-------------------
 1 file changed, 20 insertions(+), 20 deletions(-)

diff --git a/gnu/packages/simulation.scm b/gnu/packages/simulation.scm
index ab86e1d792..5341c0ae55 100644
--- a/gnu/packages/simulation.scm
+++ b/gnu/packages/simulation.scm
@@ -1204,17 +1204,6 @@ (define-public python-dolfin-adjoint
             "tests/migration/viscoelasticity/test-results")
            #t))))
     (build-system python-build-system)
-    (inputs
-     (list fenics openmpi pybind11))
-    (native-inputs
-     (list pkg-config
-           python-coverage
-           python-decorator
-           python-flake8
-           python-pkgconfig
-           python-pytest))
-    (propagated-inputs
-     `(("scipy" ,python-scipy)))
     (arguments
      `(#:phases
        (modify-phases %standard-phases
@@ -1245,17 +1234,28 @@ (define-public python-dolfin-adjoint
          ;; fails with an ImportError if it sees that the backend module
          ;; has already been loaded.
          (delete 'sanity-check))))
+    (inputs
+     (list fenics openmpi pybind11))
+    (native-inputs
+     (list pkg-config
+           python-coverage
+           python-decorator
+           python-flake8
+           python-pkgconfig
+           python-pytest))
+    (propagated-inputs
+     (list python-scipy))
     (home-page "https://www.dolfin-adjoint.org")
     (synopsis "Automatic differentiation library")
-    (description "@code{python-dolfin-adjoint} is a solver of
-differential equations associated with a governing system and a
-functional of interest.  Working from the forward model the solver
-automatically derives the discrete adjoint and tangent linear models.
-These additional models are key ingredients in many algorithms such as
-data assimilation, optimal control, sensitivity analysis, design
-optimisation and error estimation.  The dolfin-adjoint project
-provides the necessary tools and data structures for cases where the
-forward model is implemented in @code{fenics} or
+    (description
+     "@code{python-dolfin-adjoint} is a solver of differential equations
+associated with a governing system and a functional of interest.  Working from
+the forward model the solver automatically derives the discrete adjoint and
+tangent linear models.  These additional models are key ingredients in many
+algorithms such as data assimilation, optimal control, sensitivity analysis,
+design optimisation and error estimation.  The dolfin-adjoint project provides
+the necessary tools and data structures for cases where the forward model is
+implemented in @code{fenics} or
 @url{https://firedrakeproject.org,firedrake}.")
     (license license:lgpl3)))
 
-- 
2.41.0

